app/decorators/order_decorator.rb

```diff
@@ -0,0 +1,23 @@
+class OrderDecorator
+  include ActionView::Helpers::NumberHelper
+  attr_reader :model
+
+  def initialize(model)
+    @model = model
+  end
+
+  def name
+    "Order: #{model.order_number} #{model.order_status}"
```

Comment:
I18n :)
Every string of text, that user can see on front-end, should be moved to i18n